config-vm.md

```diff
@@ -0,0 +1,49 @@
+# <a name="VirtualMachineSpecificContainerConfiguration" /> Virtual-machine-specific Container Configuration
+
+Virtual-machine-based containers require additional configuration to that specified in the [default spec configuration](config.md).
+
+This **optional** configuration is specified in a "VM" object:
+
+* **`imagePath`** (string, required) host path to file that represents the root filesystem for the container virtual machine.
+* **`kernel`** (object, required) specifies details of the kernel to boot the container virtual machine with.
+* **`hypervisor`** (object, required) specifies details of the hypervisor that manages the container virtual machine.
+
+Note that `imagePath` refers to an absolute path on the host (outside of the virtual machine).
+This field is distinct from the **`path`** field in the [Root Configuration](config.md#Root-Configuration) section since in the context of a virtual-machine-based runtime:
+
+* **`imagePath`** will represent the host path to the container virtual machine root filesystem. This is the root filesystem that the virtual machine **`kernel`** will boot into.
+* The container root filesystem specified by **`path`** from the [Root Configuration](config.md#Root-Configuration) section will be mounted inside the virtual machine at a location chosen by the virtual-machine-based runtime.
+
+The virtual-machine-based runtime will use these two path fields to arrange for the **`path`** from the [Root Configuration](config.md#Root-Configuration) section to be presented to the process to run as the root filesystem.
+
+## <a name="KernelObject" /> Kernel Object
+
+Used by virtual-machine-based runtimes only.
+
+* **`path`** (string, required) specifies the host path to the kernel used to boot the container virtual machine.
+* **`parameters`** (string, optional) specifies a space-separated list of parameters to pass to the kernel.
+* **`initrd`** (string, optional) specifies the host path to an initial ramdisk to be used by the container virtual machine.
+
+## <a name="HypervisorObject" /> Hypervisor Object
+
+Used by virtual-machine-based runtimes only.
+
+* **`path`** (string, required) specifies the host path to the hypervisor used to manage the container virtual machine.
+* **`parameters`** (string, optional) specifies a space-separated list of parameters to pass to the hypervisor.
```

Comment:
If this is going to be an array, can it be a JSON array (“array of strings”) instead of a single JSON string with internal space delimiters?

Reply:
Given that this is supposed to be a parameter list that could be passed as is to the command, and would obviously be quite specific to that binary, what would be the advantage of using a JSON array?

Reply:
> Given that this is supposed to be a parameter list that could be passed as is to the command, and would obviously be quite specific to that binary...

So is [`execv`'s `argv`][1], and we use an array there.

> ... what would be the advantage of using a JSON array?

No need to quote/escape spaces within arguments and a simpler structure for removing/replacing arguments while composing the array.

[1]: http://pubs.opengroup.org/onlinepubs/9699919799/functions/exec.html